packages/tokens/src/colors.ts

```diff
@@ -0,0 +1,300 @@
+import { palette } from '@leafygreen-ui/palette';
+
+import { Mode } from './mode';
+
+const { black, blue, gray, green, red, white, yellow } = palette;
+
+const color = {
+  [Mode.Dark]: {
```

Comment:
It could be easier to read/maintain if we break out at least each dark/light mode object, or even each component level object.

e.g.
```
const darkModeBackgroundColors = {...}

const darkModeColors = {
  background: darkModeBackgroundColors,
  ...
}

const colors = {
  dark: darkModeColors
}
```

Reply:
(could even have multiple files if that makes it easier)

Reply:
Misread this at first - this seems like a good idea. I'll update.

Reply:
I think a file for each would be overkill. Maybe a file for darkmode and for light mode?